src/models/logins.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const logins = sequelize.define('logins', {
+    email: DataTypes.STRING,
+    password: DataTypes.STRING,
+    lastLogin: DataTypes.DATE
+  }, {});
+  logins.associate = function(models) {
+    // associations can be defined here
```

Comment:
No association was defined here  to reference your users model ?

Reply:
ok

Reply:
ok